libs/nghttp3/Makefile

```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nghttp3
-PKG_VERSION:=1.1.0
-PKG_RELEASE:=1
+PKG_VERSION:=1.2.0
+PKG_RELEASE:=r1
```

Comment:
This will be done by treewide commit as I told you in the other PRs.
Check this out: https://github.com/openwrt/packages/issues/23706